README.md

```diff
@@ -1,10 +1,295 @@
-# Packet tracer
+# Retis
 
 Tracing packets in the [Linux](https://kernel.org) networking stack, using
```

Comment:
@atenart, not a full review, but looks already nice.
One thing I was wondering was if it could make sense to embed some badges [0] so that the status stands out (e.g. [1], [2])

https://api.cirrus-ci.com/github/retis-org/retis.svg?task=Python%20lint%20%26%20format
https://api.cirrus-ci.com/github/retis-org/retis.svg?task=Ubuntu%20Jammy

[0] https://cirrus-ci.org/guide/writing-tasks/#badges-in-markdown
[1] https://api.cirrus-ci.com/github/retis-org/retis.svg?task=Python%20lint%20%26%20format
[2] https://api.cirrus-ci.com/github/retis-org/retis.svg?task=Ubuntu%20Jammy

Reply:
uhm, you're right. Probably not useful for the way we use them.

https://api.cirrus-ci.com/github/retis-org/retis.svg?branch=pull/169&task=Centos%209%20Stream